make/modules/java.desktop/lib/Awt2dLibraries.gmk

```diff
@@ -251,6 +257,7 @@ ifeq ($(call isTargetOs, windows macosx), false)
         DISABLED_WARNINGS_clang_aix_gtk3_interface.c := parentheses logical-op-parentheses, \
         DISABLED_WARNINGS_clang_aix_sun_awt_X11_GtkFileDialogPeer.c := parentheses, \
         DISABLED_WARNINGS_clang_aix_awt_InputMethod.c := sign-compare, \
+        DISABLED_WARNINGS_clang_screencast_pipewire.c := format-nonliteral, \
```

Comment:
move this one closer to the other clang (non-aix) suppressions

Reply:
Done.